### PR TITLE
Tweaks from template downstream review #80

### PR DESCRIPTION
**Pull request by NukeManDan**
Delta from:
- https://github.com/substrate-developer-hub/substrate-parachain-template/pull/80
- https://github.com/substrate-developer-hub/substrate-parachain-template/pull/82

### CODE REVIEW

File: parachain-template/node/src/command.rs [hunk 1]
```diff
@@ -82,8 +82,8 @@ impl SubstrateCli for RelayChainCli {
 	fn description() -> String {
 		"Parachain Collator Template\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
-		to the relaychain node.\n\n\
-		parachain-collator [parachain-args] -- [relaychain-args]"
+		to the relay chain node.\n\n\
+		parachain-collator [parachain-args] -- [relay_chain-args]"
```
Comment (KiChjang) [inline parachain-template/node/src/command.rs:86]: Shouldn't this be:

```suggestion
		parachain-collator [parachain-args] -- [relay-chain-args]"
```

File: polkadot-parachains/src/command.rs [hunk 2]
```diff
@@ -191,8 +191,8 @@ impl SubstrateCli for RelayChainCli {
 		format!(
 			"Polkadot collator\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
-		to the relaychain node.\n\n\
-		{} [parachain-args] -- [relaychain-args]",
+		to the relay chain node.\n\n\
+		{} [parachain-args] -- [relay_chain-args]",
```
Comment (KiChjang) [inline polkadot-parachains/src/command.rs:195]: Same here:

```suggestion
		{} [parachain-args] -- [relay-chain-args]",
```

**Comment by NukeManDan** [comment]
More updates based on the bump to v0.9.12 downstream now too, please re-review. :pray:

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**